bedrock/products/models.py

```diff
@@ -109,3 +113,221 @@ def get_context(self, request, *args, **kwargs):
+                ),
+                FieldPanel(
+                    "split_subheading",
+                    heading="Intro Subheading (optional)",
+                ),
+                FieldPanel(
+                    "split_content",
+                    heading="Intro Content (optional)",
+                ),
+                FieldRowPanel(
+                    [
+                        FieldPanel("split_button_text", heading="Intro Button text"),
+                        FieldPanel("split_button_link", heading="Intro Button link"),
+                    ]
+                ),
+                FieldPanel("split_image", heading="Intro Image"),
+            ],
+            heading="Intro Split Block",
+        ),
+        MultiFieldPanel(
+            [
+                FieldPanel("articles_heading"),
+                FieldPanel("articles_subheading"),
+            ],
+            heading="Article Section Heading",
+        ),
+        HelpPanel(content="The articles will be linked to here"),
+        FieldPanel("call_to_action_bottom"),
+    ]
+
+    template = "products/monitor/cms/index.html"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_context(self, request):
+        context = super().get_context(request)
+        context["articlepages"] = MonitorArticlePage.objects.live().public()  # live() and public() are wagtail filters
+        return context
+
+
+class MonitorArticlePage(AbstractBedrockCMSPage):
+    # promote panels
+    icon = models.ForeignKey(
+        "cms.BedrockImage",
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    promote_panels = AbstractBedrockCMSPage.promote_panels + [
+        FieldPanel("icon"),
+    ]
+
+    # content panels
+    subheading = models.CharField(
+        max_length=255,
+        blank=True,
+    )
+    summary = RichTextField(null=True, blank=True, features=["bold", "italic", "link", "ul", "ol"])
+    call_to_action_middle = models.ForeignKey(
+        "products.MonitorCallToActionSnippet",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    content = RichTextField(null=True, blank=False, features=["bold", "italic", "link", "ul", "ol", "h2", "h3"])
+    call_to_action_bottom = models.ForeignKey(
+        "products.MonitorCallToActionSnippet",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    content_panels = AbstractBedrockCMSPage.content_panels + [
+        FieldPanel("subheading"),
+        FieldPanel("summary"),
+        FieldPanel("call_to_action_middle"),
+        FieldPanel("content"),
+        FieldPanel("call_to_action_bottom"),
+    ]
+
+    template = "products/monitor/cms/article.html"
+
+    parent_page_types = ["MonitorArticleIndexPage"]  # must be child of MonitorArticleIndexPage
+
+    def get_context(self, request):
```

Comment:
If we can't add a test for this now, please feel free to ticket up adding one and pop my face on it. Happy to help